compose/container.py

```diff
@@ -64,7 +68,13 @@ def name(self):
 
     @property
     def name_without_project(self):
-        return '{0}_{1}'.format(self.labels.get(LABEL_SERVICE), self.number)
+        project = self.labels.get(LABEL_PROJECT)
+        service = self.labels.get(LABEL_SERVICE)
```

Comment:
We just merged a change which makes `self.service` a property. When you rebase I'd suggest using that directly.